commands/git/countcontributors.go

```diff
@@ -0,0 +1,340 @@
+package git
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/google/go-github/v56/github"
+	"github.com/jfrog/froggit-go/vcsclient"
+	"github.com/jfrog/froggit-go/vcsutils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-security/utils"
+	ioUtils "github.com/jfrog/jfrog-client-go/utils/io"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"golang.org/x/exp/maps"
+	"sort"
+	"strings"
+	"time"
+)
+
+type scmTypeName string
+
+const (
+	Github                        = scmTypeName("github")
+	Gitlab                        = scmTypeName("gitlab")
+	BitbucketServer               = scmTypeName("bitbucket")
+	DefaultContContributorsMonths = 3
+	getCommitsRetryNumber         = 5
+	GithubTokenEnvVar             = "JFROG_CLI_GITHUB_TOKEN"    // #nosec G101
```

Comment:
I think we can remove the comments at the end

Reply:
The static analysis recognizes it as a secret- so I added it so it will be ignored.